test/integration-tests/Seaport.test.ts

```diff
@@ -145,3 +147,113 @@ describe('Seaport', () => {
       .withArgs(0, customErrorSelector)
   })
 })
+
+describe('SeaportV2', () => {
```

Comment:
Optionally, we could reduce code redundancy (right now its basically copy paste from the original test suite) by looping over the original one with the new Commands.SEAPORT_V2 and the new NFT contract address. However the test where we buy two cryptoCovens for `fulfillAvailableAdvancedOrders` wouldn't work as we don't have the orders right now.